src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -761,6 +802,11 @@ public unsafe VkHandle CreateSurface<T>(VkHandle instance, T* allocator)
             return surface[0];
         }
 
+        /// <summary>
+        /// Get the extensions required for Vulkan to work on this platform.
+        /// </summary>
+        /// <param name="count">The number of extensions in the returned array</param>
+        /// <returns>An array of strings, containing names for all required extensions</returns>

```

Comment:
Was there not any documentation to inherit for this method? If not, these doc comments should be moved to the relevant interface in Windowing.Common instead